Gems/PhysX/Code/Source/PhysXCharacters/Components/CharacterGameplayComponent.cpp

```diff
@@ -35,6 +36,10 @@ namespace PhysX
                     ->DataElement(AZ::Edit::UIHandlers::Default, &CharacterGameplayConfiguration::m_gravityMultiplier,
                         "Gravity Multiplier", "Multiplier for global gravity value that applies only to this character entity.")
                     ->Attribute(AZ::Edit::Attributes::Step, 0.1f)
+                    ->DataElement(AZ::Edit::UIHandlers::Default, &CharacterGameplayConfiguration::m_groundDetectionBoxHeight,
+                        "Ground Detection Box Height", "Vertical size of box to use when testing for ground contact.")
```

Comment:
question: Where does this box start from? Would it be possible to expand the comment slightly to give a little more context?

Reply:
OK, I'll elaborate on the comment.